packages/strapi-plugin-upload/admin/src/components/UploadList/index.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { ModalBody } from 'strapi-helper-plugin';
+import { Button } from '@buffetjs/core';
+import createMatrix from '../../utils/createMatrix';
+import getTrad from '../../utils/getTrad';
+import CardImgWrapper from '../CardImgWrapper';
+import InfiniteLoadingIndicator from '../InfiniteLoadingIndicator';
+import HeaderWrapper from './HeaderWrapper';
+
+const UploadList = ({
+  filesToUpload,
+  onClickCancelUpload,
+  onGoToAddBrowseFiles,
+}) => {
+  const matrix = createMatrix(filesToUpload);
+  const filesToUploadLength = filesToUpload.length;
+  const titleId = `modal.upload-list.sub-header-title.${
+    filesToUploadLength > 1 ? 'plural' : 'singular'
+  }`;
+
+  return (
+    <>
+      <HeaderWrapper>
+        <div>
+          <div className="assets-selected">
```

Comment:
This should be <AssetsSelected> component.

Reply:
I don't really see the use of creating a dedicated component here, it's already small part of a component List so yes we could create one but it won't be reusable.